CRM/Import/Form/Summary.php

```diff
@@ -1,92 +0,0 @@
-<?php
-/*
- +--------------------------------------------------------------------+
- | Copyright CiviCRM LLC. All rights reserved.                        |
- |                                                                    |
- | This work is published under the GNU AGPLv3 license with some      |
- | permitted exceptions and without any warranty. For full license    |
- | and copyright information, see https://civicrm.org/licensing       |
- +--------------------------------------------------------------------+
- */
-
-/**
- *
- * @package CRM
- * @copyright CiviCRM LLC https://civicrm.org/licensing
- */
-
-use Civi\Api4\UserJob;
-
-/**
- * This class summarizes the import results.
- *
- * TODO: CRM-11254 - if preProcess and postProcess functions can be reconciled between the 5 child classes,
- * those classes can be removed entirely and this class will not need to be abstract
- */
-abstract class CRM_Import_Form_Summary extends CRM_Import_Forms {
-
-  /**
-   * Set variables up before form is built.
-   *
-   * @return void
-   */
-  public function preProcess() {
-    $this->assignOutputURLs();
-  }
-
-  /**
-   * Return a descriptive name for the page, used in wizard header.
-   *
-   * @return string
-   */
-  public function getTitle() {
```

Comment:
@eileenmcnaughton is this function used at all, it seems to be the only one that isn't brought across

Reply:
It's overridden when the child does this `    $this->setTitle($userJob['job_type:label']);`